src/components/Context/ContextMain.tsx

```diff
@@ -0,0 +1,43 @@
+import { createContext, ReactNode, useState } from 'react'
+interface ContextInterface {
+  genreFilter:string,
+  toggleGenreFilter?: (param: any) => void,
+  release?: string,
+  toggleRelease?: (param: any) => void,
+}
+
+const defaultState = {
+  genreFilter: 'all'
+}
+
+export const ContextMain = createContext<ContextInterface>(defaultState)
+
+type Props = {
+  children: ReactNode;
+};
+
+export const MainProvider = ({ children }: Props) => {
```

Comment:
again, `main` doesn't give any information about context. Is this Genre context or something else ?